module-4/backend/week_two.md

```diff
@@ -5,18 +5,31 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+We can define a function with using "function". Can use => now. 
 2. What's the difference between asynchronous and synchronous JavaScript? 
+Asynchronous allows multiple tasks to be executed while other querys are going on. Synchronous will not execute the next task until a query is complete.
 3. What are the four pillars of Object Oriented programming?
+Abstraction, Encapsulation, Inheritance, andPolymorphism.
 4. What are some tools available in JavaScript to help you write object oriented code?
+Webpack, VIM, Gulp. 
```

Comment:
these are tools to help us write code faster and to keep code more organized, not necessarily write more object oriented code